project-fortis-pipeline/ops/create-cluster.sh

```diff
@@ -73,12 +79,29 @@ echo "Finished. Now setting up fortis graphql service in kubernetes."
   "${cogspeechsvctoken}" \
   "${cogtextsvctoken}" \
   "${translationsvctoken}" \
-  "${fortis_site_clone_url}"
+  "${fortis_site_clone_url}" \
+  "${endpoint_protection}" \
+  "${tls_hostname}" \
+  "${tls_certificate_b64}" \
+  "${tls_key_b64}" \
+  "${lets_encrypt_email}" \
+  "${lets_encrypt_api_endpoint}"
 while :; do
-  fortis_service_ip="$(kubectl get svc project-fortis-services-lb -o jsonpath='{..ip}')"
+  if [ "${endpoint_protection}" == "none" ]; then
+  	fortis_service_ip="$(kubectl get svc project-fortis-services-lb -o jsonpath='{..ip}')"
+  else
+  	fortis_service_ip="$(kubectl get svc/nginx-ingress-controller --namespace=nginx-ingress -o jsonpath='{..ip}')"
+  fi
   if [ -n "${fortis_service_ip}" ]; then break; else echo "Waiting for project-fortis-services IP"; sleep 5s; fi
 done
-readonly graphql_service_host="http://${fortis_service_ip}"
+if [ "${endpoint_protection}" == "none" ]; then
+  readonly graphql_service_host="http://${fortis_service_ip}"
+else
+  readonly graphql_service_host="https://${tls_hostname}"
+  if [ "${endpoint_protection}" == "tls_lets_encrypt" ]; then
+  	readonly mx_record_entry="@.""$(echo ${lets_encrypt_email} | awk -F'@' '{print $2}')"
```

Comment:
Nice use of awk. `cut -d'@' -f2` might be clearer for the uninitidated.

Alternatively, you can do this in pure bash using the `#` operator ([more details](http://tldp.org/LDP/abs/html/string-manipulation.html#STRING-MANIPULATION)):

```sh
readonly mx_record_entry="@.${lets_encrypt_email#*@}"
```

Here's a little test that you can run in your console: `a="foo@bar.com"; echo "${a#*@}"`